### DISCUSSION TITLE
Looking for more info on gigapixel image refinement.

### DISCUSSION THREAD

**Discussion by trojanGoat** [NVlabs/instant-ngp#1113] [General]
Hello There,

I've seen a couple users enquiring about creating gigapixel images and the demonstrations keep mentioning this function is available.  Are there any examples I can follow?  How can I recreate the demo shown on this page?
https://nvlabs.github.io/instant-ngp/

![image](https://user-images.githubusercontent.com/19735035/205498815-5bf4317d-1eb5-46c8-bec3-a100e1dacc77.png)

**Comment by Tom94** [comment]
Hi, from the README:

> To reproduce the gigapixel results, download, for example, [the Tokyo image](https://www.flickr.com/photos/trevor_dobson_inefekt69/29314390837) and convert it to `.bin` using the `scripts/convert_image.py` script. This custom format improves compatibility and loading speed when resolution is high. Now you can run:
>
> ```sh
> instant-ngp$ ./build/testbed --scene data/image/tokyo.bin
> ```

You can find the girl with a pearl earring picture on its wikipedia page https://commons.wikimedia.org/wiki/File:Girl_with_a_Pearl_Earring_(Full_Renovation).jpg

**Comment by trojanGoat** [comment]
Thanks for the info.  So the goal of loading a single image is to replicate the original as close as possible.  Is that correct?
